envoy/tests/legacy/test_unit.py

```diff
@@ -289,7 +291,7 @@ def test_stats_prefix_optional_tags(
     standard_tags.append('endpoint:{}'.format(instance["stats_url"]))
     tags_prefix = standard_tags + additional_tags
     c = check(instance)
-    mock_http_response(file_path=fixture_path(fixture_file)).return_value
+    mock_http_response(file_path=fixture_path(fixture_file))
```

Comment:
did we mean to remove this?

Reply:
tests were failing because of this, as it is not a useful statement